src/abstractdataframe/reshape.jl

```diff
@@ -417,7 +417,7 @@ with name specified by `dest_namescol`.
 # Arguments
 - `df` : the `AbstractDataFrame`
 - `src_namescol` : the column that will become the new header.
-  This column's element type must be `AbstractString` or `Symbol`.
+  This column's element type must be `AbstractString`, `Symbol`, or `Int`.
```

Comment:
really `Int` is allowed as `eltype`? I thought not.

Reply:
Oh, I was thinking of `dest_namescol` :facepalm: